cruise.umple/src/Generator_CodeJava.ump

```diff
@@ -1934,7 +1851,112 @@ class JavaGenerator
     }            
   }
   
+  private void prepareCodeInjection(UmpleClass aClass)
+  {
+  	prepareCodeInjectionForAttributes(aClass);
+  	prepareCodeInjectionForAssociations(aClass);
+  }
   
+  private void prepareCodeInjectionForAttributes(UmpleClass aClass)
+  {
+  	for(Attribute av : aClass.getAttributes())
+    {
+      prepareCodeInjectionForAttribute(aClass, av);
+    }
+  }
```

Comment:
Can you ensure all indentation are spaces (2 for an indent).  I am seeing some tabs in this PR